src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+
+## Leer het bij: Code, Code, Code!
+
+In het begin is het niet te overzien; verschillende tools, talen & frameworks; Vue, Angular, React etc!
+Ook worden er veel fancy Engelse namen en code talen gebruikt in dagelijks communicatie. Afhankelijk van je doel is de learning curve best stijl, zeker als je alleen een design achtergrond hebt. Er zijn ook veel overeenkomsten, die zie je vooral terug in HTML/CSS. Denk aan eigenschappen van elementen, werkstructuur en het uiteindelijke doel; gave dingen maken!
+
+Ik heb gemerkt dat het iets is wat je over je heen moet laten komen. Veel stof is logischer en minder ingewikkeld dan hoe het er in eerste instantie uitziet. Als designer heb je ook te maken met projectmatig werken, het einddoel waarborgen en een professioneel product afleveren. De tooling is alleen anders.
+Het moeilijkste om te leren is het denken als een developer; een soort analytische, oplossingsgerichte visie die je moet hebben, naast je algemene programmeer vaardigheden natuurlijk. Met genoeg oefening is dit zeker onder de knie te krijgen en moeten de meeste designers met een beetje geduld, hier wel uit gaan komen.
+
+<figure>
+    <img src="/assets/design-het-web/code.jpg" alt="Code">
+    <figcaption>Leer project HTML/CSS</figcaption>
+</figure>
+
+## De dubbele pet: WordPress
+
+In mijn geval was ik meer vaardig in design terwijl ik meer interesse had voor de development. Als tussenoplossing daarvoor maakte ik gebruik van WordPress.
+Een veel gebruikt CMS die theme based veel mogelijkheden biedt. Na het maken van het design kon ik het in een populaire builder zoals [Avada](https://avada.theme-fusion.com/) of [Elementor](https://elementor.com/) het design ongeveer nabouwen. Het design van het web is voor klein MKB veruit het belangrijkst. De Achterliggende techniek is niet zo belangrijk. Door een behoorlijke groep mensen word de designer ook gezien als developer. Als designer heb je dus een dubbele pet als je het hebt over Theme based werken.
+
+De technische kennis beperkt zich tot basis HTML/CSS vaardigheden wat goed te overzien is. Een groot nadeel van WordPress is, is dat het gebruik maakt van plug-in’s en dat er veel updates gaande zijn. Op het moment dat er hier ergens wat fout gaat ben je afhankelijk van je technische kennis of de bouwer van het thema/de plug-in.
+
+<figure>
+    <img src="/assets/design-het-web/avada.jpg" alt="Avada">
+    <figcaption>Avada website 2022</figcaption>
+</figure>
+
+## Sluit het uit: No-Code
+
+No-code frameworks zijn programma’s waarmee niet-technische mensen software kunnen ontwikkelen zonder te coderen. Deze tools hebben meestal een gebruiksvriendelijke UI en dingen als een drag-and-drop-mogelijkheid, zodat er makkelijk assets toegevoegd kunnen worden zonder technische skills. Dus wil je ver van de code vandaan blijven? Dit is je beste optie.
```

Comment:
> Dus wil je ver van de code vandaan blijven? Dit is je beste optie.

Isn't the other option to actually be a designer instead of going `no-code`? 😁